src/vs/code/electron-browser/sharedProcess.js

```diff
@@ -0,0 +1,102 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Seems like a lot of overlap with https://github.com/Microsoft/vscode/blob/master/src/vs/workbench/electron-browser/bootstrap/index.js#L48, maybe better to have a common base that both can reuse?